packages/salesforcedx-utils-vscode/src/telemetry/configUtil.ts

```diff
@@ -52,10 +52,12 @@ export class ConfigUtil {
           return localValue;
         }
       } catch (err) {
-        TelemetryService.getInstance().sendException(
-          'get_config_value_local',
-          err.message
-        );
+        if (err instanceof Error) {
```

Comment:
same comment as before, after converting those three instances, we probably don't need to check if err is an instance of Error

Reply:
agreed. see https://github.com/forcedotcom/salesforcedx-vscode/pull/4433#discussion_r976036460